### PR TITLE
Doc: update guides

### PR DESCRIPTION
**Pull request by TheDZhon**
### Update tokens integration guide
- [x] Update well-known integration references (include Mellow LRT)
- [x] Update token pricing approach
- [x] Update liquidity venues
- [x] Update risks
- [x] More L2 networks available
- [x] Mention SDK 

###  Update bridging guide
- [x] Exclude Lido contributors commitment from reviewing
- [x] Propose `state-mate`
- [x] Refer to Mode
- [x] Add Scroll case

### CODE REVIEW

File: docs/guides/lido-tokens-integration-guide.md [hunk 1]
```diff
@@ -378,54 +448,55 @@ To claim ether you need to call:
 
 stETH/wstETH as DeFi collateral is beneficial for several reasons:
 
-- stETH/wstETH is almost as safe as ether, price-wise: barring catastrophic scenarios, its value tends to hold the ETH peg well;
+- stETH/wstETH is almost as safe as ether, price-wise: barring catastrophic scenarios, its value tends to hold the ETH 1:1 well;
 - stETH/wstETH is a productive token: getting rewards on collateral effectively lowers the cost of borrowing;
-- stETH/wstETH is a very liquid token with billions of liquidity locked in liquidity pools ([Curve](https://curve.fi/steth), [Balancer v2](https://app.balancer.fi/#/pool/0x32296969ef14eb0c6d29669c550d4a0449130230000200000000000000000080))
+- stETH/wstETH is a very liquid token with billions of liquidity locked in liquidity pools (see [above](./lido-tokens-integration-guide.md#sttokens-steth-and-wsteth))
```
Comment (arwer13) [inline docs/guides/lido-tokens-integration-guide.md:453]:
```suggestion
- stETH/wstETH is a very liquid token with billions of liquidity locked in liquidity pools (see [above](#sttokens-steth-and-wsteth))
```

File: docs/guides/lido-tokens-integration-guide.md [hunk 2]
```diff
@@ -378,54 +448,55 @@ To claim ether you need to call:
...
 
 - On Maker, [wstETH collateral (scroll down to Dai from WSTETH-A section)](https://daistats.com/#/collateral) can be used to mint DAI stablecoin. See [Lido's blog post](https://blog.lido.fi/makerdao-integrates-lidos-staked-eth-steth-as-collateral-asset/) for more details.
-- On AAVE, multiple tokens can be [borrowed against stETH](https://app.aave.com/reserve-overview/?underlyingAsset=0xae7ab96520de3a18e5e111b5eaab095312d7fe84&marketName=proto_mainnet). See [Lido's blog post](https://blog.lido.fi/aave-integrates-lidos-steth-as-collateral/) for more details. Please note: stETH is only supported on AAVE as lending collateral. Borrowing stETH on AAVE is not currently supported. However, any asset can be borrowed on AAVe via a flashloan. Due to a known [1-2 wei corner case](./lido-tokens-integration-guide#1-2-wei-corner-case, there's a certain situation when a flashloan transaction can revert. Please visit [stETH on AAVE caveats](https://docs.lido.fi/token-guides/steth-on-aave-caveats) article for more details.
+- On AAVE v3, multiple tokens can be borrowed against wstETH on various chains (see the list of the [markets](./lido-tokens-integration-guide.md#sttokens-steth-and-wsteth))
```
Comment (arwer13) [inline docs/guides/lido-tokens-integration-guide.md:458]:
```suggestion
- On AAVE v3, multiple tokens can be borrowed against wstETH on various chains (see the list of the [markets](#sttokens-steth-and-wsteth))
```

**Review by arwer13 — CHANGES_REQUESTED** [review]

**Comment by TheDZhon** [comment]
Thanks, @arwer13 
Applied all of the suggestions :+1:

File: docs/guides/lido-tokens-integration-guide.md [hunk 3]
```diff
@@ -49,46 +84,54 @@ A non-fungible token (NFT) is used to represent a withdrawal request position [i
...
-The accounting oracle has sanity checks on both max APR reported (the APR cannot exceed 27%, which means a daily rebase is limited to `27/365%`) and total staked amount drop (staked ether decrease reported cannot exceed 5%). Currently, the oracle report is based on five oracle daemons hosted by established node operators selected by the DAO. As soon as five out of nine oracle daemons report the same data, reaching the consensus, the report goes to the Lido smart contract, and the rebase occurs.
+Normally, stETH rebases happen daily when the Lido oracle reports the Consensus Layer ether balance update. The rebase can be positive or negative, depending on the validators' performance. In case Lido's validators get slashed or penalized, the stETH balances can decrease according to penalty sizes. However, daily rebases have never been negative by the time of writing.
```
Comment (mymphe) [inline docs/guides/lido-tokens-integration-guide.md:117]: > slashed or penalized

Same thing?

File: docs/guides/lido-tokens-integration-guide.md [hunk 4]
```diff
@@ -49,46 +84,54 @@ A non-fungible token (NFT) is used to represent a withdrawal request position [i
...
+
+The accounting oracle has sanity checks on both max APR reported (the APR cannot exceed 27%, which means a daily rebase is limited to `27/365%`) and total staked amount drop (staked ether decrease reported cannot exceed 5%).
```
Comment (mymphe) [inline docs/guides/lido-tokens-integration-guide.md:119]: `(27/365)%`

File: docs/guides/lido-tokens-integration-guide.md [hunk 5]
```diff
@@ -49,46 +84,54 @@ A non-fungible token (NFT) is used to represent a withdrawal request position [i
...
+
+Currently, the oracle report is based on five oracle daemons hosted by established node operators selected by the DAO. As soon as five out of nine oracle daemons report the same data, reaching the consensus, the report goes to the Lido smart contract, and the rebase occurs.
```
Comment (mymphe) [inline docs/guides/lido-tokens-integration-guide.md:121]: > report is based on five oracle daemons

It's based on nine, but at least five need to report, isn't it? Better rephrase completely, e.g.

Oracle network includes 9 oracles...

File: docs/guides/lido-tokens-integration-guide.md [hunk 6]
```diff
@@ -378,54 +448,58 @@ To claim ether you need to call:
...
 
 ### Wallet integrations
 
-Lido's Ethereum staking services have been successfully integrated into the most popular DeFi wallets, including Ledger, MyEtherWallet, ImToken and others.
+Lido's Ethereum staking services have been successfully integrated into the most popular DeFi wallets, including Ledger, Metamask, MyEtherWallet, ImToken and others.
 Having stETH integrated can provide wallet users with a great user experience of direct staking from the wallet UI itself.
 
-Lido DAO runs a referral program rewarding wallets and other apps for driving liquidity to the Lido staking protocol. At the moment, the referral program is in [whitelist mode](https://research.lido.fi/t/switch-referral-program-to-whitelist-mode/1014). Please contact Lido bizdev team to find out if your wallet might be eligible for referral program participation.
+When adding stETH support to a DeFi wallet, it is important to preserve stETH's rebasing nature.
+Avoid storing cached stETH balance for extended periods of time (over 24 hours), and keep in mind it doesn't necessarily take a transaction to change stETH balance.
```
Comment (mymphe) [inline docs/guides/lido-tokens-integration-guide.md:473]: > it doesn't necessarily take a transaction to change stETH balance

It does take a transaction, though not the holder's but still haha

We can rephrase it to something like
"Note that stETH balance changes on each rebase without any incoming or outgoing user transfers and does not emit ERC-20 'Transfer' events"

File: docs/guides/lido-tokens-integration-guide.md [hunk 7]
```diff
@@ -378,54 +448,58 @@ To claim ether you need to call:
...
-3. Slashing risk.
-Beacon chain validators risk staking penalties, with up to 100% of staked funds at risk if validators fail. To minimize this risk, Lido stakes across multiple professional and reputable node operators with heterogeneous setups, with additional mitigation in the form of self-coverage.
-4. stETH price risk.
-Users risk an exchange price of stETH which is lower than inherent value due to withdrawal restrictions on Lido, making arbitrage and risk-free market-making impossible. The Lido DAO is driven to mitigate the above risks and eliminate them to the extent possible. Despite this, they may still exist and, as such, we must communicate them.
+There exist a number of potential risks when staking using liquid staking protocols.
+
+### Smart contract security
+
+  There is an inherent risk that Lido could contain a smart contract vulnerability or bug. The Lido code is open-sourced, audited, and covered by an extensive bug bounty program to minimize this risk. To mitigate smart contract risks, all of the core Lido contracts are audited. Audit reports can be found here. Besides, Lido is covered with a massive Immunefi bug bounty program.
```
Comment (mymphe) [inline docs/guides/lido-tokens-integration-guide.md:497]: > open-sourced

open-source

"Open-sourced" sounds like it was "sourced from the open"  🌚

File: docs/token-guides/wsteth-bridging-guide.md [hunk 8]
```diff
@@ -317,6 +340,9 @@ Please, deploy to Holesky if possible because Holešky has better long-term expo
 Yes, sure. For example, [OptimismBridgeExecutor](https://github.com/lidofinance/governance-crosschain-bridges/blob/master/contracts/bridges/OptimismBridgeExecutor.sol) has been [reused](https://basescan.org/address/0x0E37599436974a25dDeEdF795C848d30Af46eaCF#code) on Base network.
 If so, please don't alter the contract's code and use the same names. It allows to keep the audit valid and track origins.
 
+To speed the process up, you might perform a deployment verification against the bytecode already used for another network and configuration/storage state comparison to be 1:1 except only for the network specific configuration changes needed.
```
Comment (mymphe) [inline docs/token-guides/wsteth-bridging-guide.md:343]: > To speed the process up

To speed up the process

File: docs/guides/lido-tokens-integration-guide.md (hunk 3 above)
Comment (TheDZhon) [reply docs/guides/lido-tokens-integration-guide.md:117]: nor really

**Review by mymphe — APPROVED** [review]

**Pull request merged**